proposals/78:upgrade-17/agd-tools.js

```diff
@@ -0,0 +1,208 @@
+import {
```

Comment:
sigh. another copy of these tools. Here's hoping to migrate stuff to synthetic-chain